src/main/java/gmbs/controller/CarRaceOperator.java

```diff
@@ -0,0 +1,70 @@
+package gmbs.controller;
+
+import gmbs.model.RacingCars;
+import gmbs.model.RandomNumberGenerator;
+import gmbs.model.RepetitionNumber;
+import gmbs.view.Display;
+import gmbs.view.UserInput;
+
+import java.util.Arrays;
+import java.util.List;
+
+public class CarRaceOperator {
+
+    private static final Display display = new Display();
+    private static final UserInput userInput = new UserInput();
+    private static final String SPLIT_VALUE = ",";
+    private static final RandomNumberGenerator generator = new RandomNumberGenerator(0, 9);
+
+    private RacingCars racingCars;
+
+    private int scanRepetition() {
+        int repetition;
+        try {
+            repetition = Integer.parseInt(new RepetitionNumber(userInput.userInput()).number());
+        } catch (IllegalArgumentException e) {
+            System.out.println(e.getMessage());
+            repetition = scanRepetition();
+        }
+        return repetition;
+    }
+
+    private List<String> splitInput(String userInput) {
+        return Arrays.asList(userInput.split(SPLIT_VALUE));
+    }
+
+    private List<String> requestCarNames() {
+        List<String> carNames;
+        try {
+            carNames = splitInput(userInput.userInput());
+        } catch (IllegalArgumentException e) {
+            System.out.println(e.getMessage());
+            carNames = requestCarNames();
+        }
+        return carNames;
+    }
```

Comment:
```suggestion
    private List<String> requestCarNames() {
        try {
            return splitInput(userInput.userInput());
        } catch (IllegalArgumentException e) {
            System.out.println(e.getMessage());
            return requestCarNames();
        }
    }
```
위와 같이 변경하면 더 깔끔한 것 같은데 어떻게 생각하세요~?

Reply:
좋은 방법인것 같습니다

depth를 줄이기 위해 재귀를 통하여 함수를 호출하는 방법은 어떻게 생각하시나요
재귀적 호출을  지양하는 것이 좋다고 생각하는데 미션의 요구사항을 지키기 위해서 부득이하게 이용햇습니다
다른 좋은 방법을 찾아봐야겠어요